Thumbnail/ConsumerThumbnail.php

```diff
@@ -95,10 +95,7 @@ public function generate(MediaProviderInterface $provider, MediaInterface $media
 
             // BC compatibility for missing EventDispatcher
             if (null === $this->dispatcher) {
-                trigger_error(
-                    'Since version 2.3.3, passing an empty parameter in argument 4 for __construct() in '.__CLASS__.' is deprecated and the workaround for it will be removed in 3.0.',
-                    E_USER_DEPRECATED
-                );
+                trigger_error('Since version 2.3.3, passing an empty parameter in argument 4 for __construct() in '.__CLASS__.' is deprecated and the workaround for it will be removed in 3.0.', E_USER_DEPRECATED);
```

Comment:
Yeah but he did not actually mean use `git revert`, he meant "just make it disappear from the commits".